package.json

```diff
@@ -33,11 +32,11 @@
   "devDependencies": {
     "JSONStream": "^1.3.4",
     "async": "^2.6.1",
-    "babel-cli": "^6.26.0",
-    "babel-eslint": "^8.2.6",
-    "babel-plugin-add-module-exports": "^0.2.1",
-    "babel-plugin-transform-runtime": "^6.23.0",
```

Comment:
I've removed transform-runtime since it seems to be mostly useful when building to very old Node versions and using it causes https://github.com/babel/babel-loader/issues/152